include/customizer/cell_customizer.hpp

```diff
@@ -0,0 +1,148 @@
+#ifndef OSRM_CELLS_CUSTOMIZER_HPP
+#define OSRM_CELLS_CUSTOMIZER_HPP
+
+#include "partition/cell_storage.hpp"
+#include "partition/multi_level_partition.hpp"
+#include "util/binary_heap.hpp"
+
+#include <boost/thread/tss.hpp>
+#include <unordered_set>
+
+namespace osrm
+{
+namespace customizer
+{
+
+class CellCustomizer
```

Comment:
This class does not maintain any state right now. Do you think we could just split it into free functions?